packages/web-components/src/components/va-radio-option/va-radio-option.scss

```diff
@@ -1,15 +1,40 @@
-@import '../../mixins/inputs.css';
+@use 'uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors';
+@use 'usa-radio/src/styles/usa-radio';
 
 :host {
   display: block;
-  margin-top: 1rem;
 }
 
 :host(:focus) {
   outline: none !important;
 }
 
-label {
+:host(:focus) label.usa-radio__label::before {	
+  outline: 2px solid var(--color-gold-light);	
+  outline-offset: 4px;	
+}
+
+:host([disabled='true']:focus) label.usa-radio__label::before {	
+  outline: none;	
+}
+
+:host([uswds]) .usa-radio__label {
+  line-height: 1.5;
+}
+
+.usa-radio__label::before {
+  box-shadow: rgb(113, 118, 122) 0px 0px 0px 2px;
```

Comment:
Why do we need to do this?

Reply:
We have a different radio button border color (Sketch link in the description): 

USWDS v3:
![Screen Shot 2022-11-22 at 1 02 54 PM](https://user-images.githubusercontent.com/872479/203399350-d03a8b59-cf8f-4736-872e-49081f4a2823.png)

VA:
![Screen Shot 2022-11-22 at 1 02 33 PM](https://user-images.githubusercontent.com/872479/203399351-1cbbe492-d959-44bf-aaa1-d8561ca927b8.png)

Reply:
I don't think we should. When we see a discrepancy like this it should be raised with Natalie. Not sure why we'd deviate here.

Reply:
Ok I'll remove it. There's also the line height difference too you can see in the screenshot above: https://github.com/department-of-veterans-affairs/component-library/blob/74aa2b344e2596394832f358485abd1f5c76807b/packages/web-components/src/components/va-radio-option/va-radio-option.scss#L21-L23

@GnatalieH The USWDS radio default border color is `#1b1b1b` (sketch has `#71767A`) and the default line-height for the radio item text is `line-height: 1.3` (sketch has 1.5).

Reply:
Currently I have the line-height 1.5 (24px) CSS override removed but if we want to add it back, the development burden is pretty low. It's just the CSS below. 

Based on Natalie's reasoning above, do we want to make this an exception @humancompanion-usds? 

```css
 .usa-radio__label { 
   line-height: 1.5; 
 } 
```

Reply:
@jamigibbs I checked the Chromatic link and all looks great! The only thing I'd change is "Id" usage to "ID usage". Thanks!

Reply:
Thanks @GnatalieH! I think we need to display it as "Id" because of the way Storybook parses the story titles. If it was "ID" it would automatically add a space between the letters so it would say "I D Usage" instead. 😞

Reply:
@GnatalieH Disregard my last comment about "ID". It looks like it's smart enough to parse this correctly. I'll update! 👍